### ISSUE TITLE
123

### ISSUE THREAD

**Issue by stevelibtcc** [stevelibtcc/test#1]
## What we have done well?
1. Experienced how to develop an exchange platform in a week
体验到了一个月做出一个交易所的感受
2. Use docker to improve the Ops and continuous deployment, we need only one person instead of 3, it let us monitor and scale-out our product more easily.
用docker改进了运维和持续发布流程， 别的团队三个人的工作我们只要一个人，还改进了监控和横向扩展能力
3. Zhujie finished 2 wallets within half a month: eth & btc.
朱杰半个月完成了两个钱包，eth和btc
4. Jackie build the website that adapts to different screen-size.
Jackie一次做完跨平台的交易所体验，以前在公司项目中没有
5. Positive work attitude, members have ownership to the project.
Team责任心比较强，有目标有ownership

##What we have done wrong?
1.	The retrospect is delayed for too long. 
等了太久才做回顾，对时间估计太乐观
2.	Bobby was too optimistic on ETA, not enough coders. 
时间估计太乐观是Bobby的错，人数分配也不够，
3. Frank gave an irresponsible “yes”.
Frank错误地答应了Bobby的请求
4. Didn’t stick to the QA process. No process designed, no unit test.
没有坚持进行好测试，没有设计流程，没有单元测试
5. Frank had high expectations to the leaving employee. 
Frank对即将离职的员工期待太高
6. Aaron and Frank didn’t make the decision to postpone.
Aaron和Frank没有做出延后发布的决定
7. To many functions to be finished. 
设想功能点太多
8. Requirement is given only after all stakeholders agree on it.
作为一个产品，要统一意见再出需求，需求变动太大
9. Aaron and Frank made 2 major changes without sufficiently communicating with Bobby.
Aaron和Frank没有和Bobby沟通好具体需求导致临上线之前做了两次大改变
10. Didn’t stick to security works, safety test is not carried through, no design for the system to defend itself against hacks.
没有坚持做安全工作，安全测试不到位，没有为系统预防入侵
11. When discuss upon a requirement, reach the details.
需求讨论的时候没有到具体细节
12. Didn’t have KY’s support to ensure volume after launch
上线之后没有和KY保证保持交易量
13. Didn’t plan the testing for systems from other teams. Such as trade engine and log-in from BTCC.
不是Dax做的系统，没有计划进行测试，比如交易引擎和BTCC的登入系统
14. Plan work-load based on regular working schedule, which Zeyu found difficult. Plan of the week should be transparent to everyone, work overtime is dangerous….
计划好一天的工作量， 但是Zeyu认为估计起来很难， 要了解一周的大进度
加班有人¬¬身安全性风险
15. Not familiar with BTCC project, should have learnt experience from BTCC>
不了解之前的BTCC项目，没有借鉴经验没有提前了解坑
16. UI is not good
UI设计比较仓促·

##How do we improve in the next scrum?
1. Use the local developer’s environment, so that every developer can 
积极使用本地开发环境，达到可以同一个人开发并且看测试结果的效果
3.	Encourage a healthy way of work. Work overtime is not sustainable and can
lead to low-quality code.
倡导健康工作方式，不要依赖加班， 因为加班赶出来的代码质量低
4. Stick to the principle, don’t rush on time.
不接受赶时间的工作方式，原则要定好坚持
5. PO needs to know when to say no to Bobby.
作为PO，在必要的时候要拒绝Bobby的需求
6. Increase efficiency with prioritization.
通过优先级来最大化效率
7. Leveled access to Log
要细分日志权限
